code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -107,10 +107,14 @@
 		to_chat(user, "<span class='notice'>\The [src] is empty.</span>")
 		return
 
+	var/i = 0
 	for(var/obj/item/ammo_casing/shotgun/loaded_shell in src) //This feels like a hack. don't code at 3:30am kids!!
 		loaded_shell.forceMove(get_turf(src))
+		loaded_shell.pixel_x = -3 + (i*4)
```

Comment:
All pixel_x and pixel_y modications should consider the pixel multiplier correctly.